### PR TITLE
Remove teacher view on /courses page

### PR DESCRIPTION
**Pull request by TurnerRiley**
With the launch of the /teach page, we no longer have use for /courses?view=teacher so this PR makes the /courses page show what is effectively the student view for any user.

### Student view
1_student
2_student
3_student

### Teacher view
1_teacher
2_teacher
3_teacher

## Links
Jira ticket: [here](https://codedotorg.atlassian.net/jira/software/c/projects/ACQ/boards/64?modal=detail&selectedIssue=ACQ-702&assignee=60d62161f65054006980bd71)

## Testing story
Local testing.

## PR Checklist:
- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/studioHomepages/Courses.jsx [hunk 1]
```diff
@@ -101,30 +78,18 @@ class Courses extends Component {
...
+                <SpecialAnnouncement />
                 <CoursesStudentEnglish />
               </div>
             )}
 
             {/* Non-English */}
             {!isEnglish && (
               <CourseBlocksIntl
-                isTeacher={isTeacher}
+                isTeacher={false}
```
Comment (TurnerRiley) [inline apps/src/templates/studioHomepages/Courses.jsx]: `isTeacher` is required for `CourseBlocksIntl` so it is defaulting to the student view here

File: apps/src/templates/studioHomepages/Courses.jsx [hunk 2]
```diff
@@ -41,41 +36,23 @@ class Courses extends Component {
...
-    } = this.props;
+    const {isEnglish, isSignedOut, modernElementaryCoursesAvailable} =
+      this.props;
 
     const {headingText, subHeadingText, description, buttonText} =
       this.getHeroStrings();
 
     // Verify background image works for both LTR and RTL languages.
-    const backgroundUrl = isTeacher
-      ? '/shared/images/banners/courses-hero-teacher.jpg'
```
Comment (dmcavoy) [inline apps/src/templates/studioHomepages/Courses.jsx:77]: Is the banner image from the teach page used anywhere else? should we remove it?

File: apps/src/templates/studioHomepages/Courses.jsx (hunk 1 above)
Comment (dmcavoy) [reply apps/src/templates/studioHomepages/Courses.jsx]: Where else is CourseBlocksIntl used? Could we update it?

Comment (TurnerRiley) [reply apps/src/templates/studioHomepages/Courses.jsx]: Only used here! I'll update it

File: apps/src/templates/studioHomepages/Courses.jsx (hunk 2 above)
Comment (TurnerRiley) [reply apps/src/templates/studioHomepages/Courses.jsx:77]: Yep you're right it's not used anywhere else! Removed it!

File: apps/src/templates/studioHomepages/CourseBlocks.jsx [hunk 3]
```diff
@@ -208,10 +208,14 @@ export class CourseBlocksHoc extends Component {
...
 export class CourseBlocksIntl extends Component {
   static propTypes = {
-    isTeacher: PropTypes.bool.isRequired,
+    isTeacher: PropTypes.bool,
     showModernElementaryCourses: PropTypes.bool.isRequired,
   };
 
+  static defaultProps = {
+    isTeacher: false,
+  };
```
Comment (megcrenshaw) [inline apps/src/templates/studioHomepages/CourseBlocks.jsx]: nit: isTeacher will be undefined if it's not passed in, which will evaluate to false in most cases. Could probably delete this, but not worth a Drone re-run

**Review by megcrenshaw — APPROVED** [review]
Could you update the index method in `courses_controller.rb` file as well to remove the `@is_teacher = (current_user&.teacher?) || params[:view] == 'teacher'` line? We don't need to pass it through any more it looks like.

Other than that, looks great! Wahoo simpler code 🥳

**Pull request merged**